Sources/Commands/Utilities/TestingSupport.swift

```diff
@@ -177,7 +171,7 @@ enum TestingSupport {
             // execution but is required when the tests are running in parallel as
             // SwiftPM repeatedly invokes the test binary with the test case name as
             // the filter.
-            let codecovProfile = buildParameters.buildPath.appending(components: "codecov", "default%m.profraw")
+            let codecovProfile = buildParameters.buildPath.appending(components: "codecov", "\(library)%m.profraw")
```

Comment:
Can this use `buildParameters.codeCovPath` as the base?

Reply:
Presumably. This bit's not new though and I'd be tempting fate changing it in this PR. We can look at a change separately.